bundles/org.openhab.binding.lcn/src/main/resources/ESH-INF/thing/thing-types.xml

```diff
@@ -658,4 +659,17 @@
 		<label>Remote Control Low Battery</label>
 		<event/>
 	</channel-type>
+
+	<channel-group-type id="hostcommands">
+		<label>Host Command (to openHAB)</label>
+		<channels>
+			<channel typeId="sendKeys" id="sendKeys"/>
+		</channels>
+	</channel-group-type>
+
+	<channel-type id="sendKeys" advanced="true">
+		<kind>trigger</kind>
+		<label>Send Keys</label>
+		<event/>
```

Comment:
Is it necessary to include `<event/>`?

Reply:
I think so, as the trigger contains payload like "A1:HIT" or "B2:MAKE". I added the options tag.